docs/pages/build-reference/variables.md

```diff
@@ -146,14 +146,18 @@ When you define environment variables on build profiles in **eas.json**, they wi
 In **app.config.js**, we may be using the API URL like this:
 
 ```js
-export default {
+export default ({ config }) => {
```

Comment:
Please add an additional snippet that demonstrates this code block rather than overwriting the minimum viable example.